src/js/components/aboutTheData/AboutTheDataPage.jsx

```diff
@@ -47,7 +47,7 @@ const AboutTheDataPage = ({
     history
 }) => {
     const { fy: urlFy, period: urlPeriod } = useParams();
-    const [dataAsOf, submissionPeriods] = useLatestAccountData();
+    const [, submissionPeriods, { year: latestFy, period: latestPeriod }] = useLatestAccountData();
```

Comment:
```suggestion
    const [submissionPeriods, { year: latestFy, period: latestPeriod }] = useLatestAccountData();
```

Reply:
So to restructure an array, we have to do this I think

```javascript
   const [first, second, third] = [1,2,3]
   const [, second, third] = [1,2,3]
```

Reply:
Ah gotcha I was thinking it was leftover from a find and replace!